.travis/osx..install.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-PACKAGES="cmake pkg-config fftw libogg libvorbis libsndfile libsamplerate jack sdl stk portaudio node fltk"
+PACKAGES="cmake pkgconfig fftw libogg libvorbis lame libsndfile libsamplerate jack sdl libgig libsoundio stk portaudio node fltk"
```

Comment:
@PhysSong is right, it's about the `for loop`.

It's because homebrew fails if we try to install a package which is already there. (Because of `set -e`).

So, first we remove already existing packages from the list of packages to install.
Unfortunately, it won't work with the wrong alias. Because, when we ask homebrew the list of installed packages, it tells pkg-config and not pkgconfig. So we don't remove it from the list, so we try to install it but it's already there, so homebrew fails...

To avoid this, I opened an issue on homebrew a few time ago, but people closed it without providing what I would consider a good fix. So... we should use `pkg-config`.